### PR TITLE
ff-HSM: PR complete, ready to merge

### PR DESCRIPTION
**Pull request by FFincannon**

### CODE REVIEW

**Comment by FFincannon** [comment]
This PR contains  formatting fixes for WebAuthn as well as the YubiHSM2 Win Deployment content

File: content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Backup_Key_Material.adoc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+
+
+== Backup Key Material
```
Comment (klali) [inline content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Backup_Key_Material.adoc]: Should this file really be empty? If we're not ready for it yet maybe we should just exclude it.

File: content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Configure_the_Primary YubiHSM_2_Device.adoc [hunk 2]
```diff
@@ -0,0 +1,175 @@
+
+
```
Comment (klali) [inline content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Configure_the_Primary YubiHSM_2_Device.adoc]: I'd prefer that we don't start with empty lines, rather have the heading on first line.

File: content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/.conf.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+  "order": [
+    "Introduction",
+    "Prerequisites_and_Preparations",
+    "Install_the_YubiHSM_Tools_and_Software"
```
Comment (klali) [inline content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/.conf.json]: there's a comma missing at the end of line here, leading to the build failing

File: content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Configure_the_Primary YubiHSM_2_Device.adoc [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+        version=1.0.0
+        pid=*
+        address=192.168.100.252
+        port=12345
+
+*Step 2*: In the Command Prompt, navigate to a directory for which you have write access and run the YubiHSM Setup with the argument `ksp`.
+
+        $ yubihsm-setup ksp
+
+_Note: When replacing the default authentication key using this setup, the new authentication key will only be operable in the same domain as the asymmetric key. This is due to the https://developers.yubico.com/YubiHSM2/Concepts/Domain.html[domain concept] that is used to compartmentalize the YubiHSM 2 device._
```
Comment (klali) [inline content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Configure_the_Primary YubiHSM_2_Device.adoc]: It's good if the link to Domain.html is relative here, just put it as the relative link "link:../../Concepts/Domain.adoc" and it'll work both in asciidoc mode and html.

File: content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Configure_the_Primary YubiHSM_2_Device.adoc [hunk 5]
```diff
@@ -0,0 +1,175 @@
...
+
+2. Start the configuration process.
+
+3. Authenticate to the YubiHSM 2 device.
+
+4. Specify whether you need to add RSA decryption.
+
+5. Enter the name of the domain in which you need the application authentication key and audit key to be available in.
+
+6. Create the wrap key. The wrap key is very important as it allows you to export and import objects from and to the device. For example, you would export and import objects for backup purposes, as described in https://Backup_Key_Material.html[Backup Key Material].
```
Comment (klali) [inline content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Configure_the_Primary YubiHSM_2_Device.adoc]: the link for backuo key material should be "link:Backup_Key_Material.adoc" to link correctly (the file is also empty).

File: content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Install_the_YubiHSM_Tools_and_Software.adoc [hunk 6]
```diff
@@ -0,0 +1,60 @@
+
+
```
Comment (klali) [inline content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Install_the_YubiHSM_Tools_and_Software.adoc]: Please remove the initial blank lines.

File: content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Install_the_YubiHSM_Tools_and_Software.adoc [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+*Step 2* On your Windows system, run both installers:
+
+         * yubihsm-cngprovider-windows-amd64.msi (YubiHSM Key Storage Provider)
+         * yubihsm-connector-windows-amd64.msi (YubiHSM Connector for Windows)
+
+
+=== Default YubiHSM 2 Default Device Configuration
+
+The YubiHSM 2 device comes with a default, factory-installed authentication key that has the password `password`. As part of the configuration in this guide, the default authentication key will be destroyed. If the device is reset to its default configuration, any objects stored on the device that are not factory-installed are destroyed. Reset instructions can be found in https://developers.yubico.com/YubiHSM2/Usage_Guides/Factory_reset.html[Factory Reset].
```
Comment (klali) [inline content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Install_the_YubiHSM_Tools_and_Software.adoc]: Please make this a relative link of "link:../Factory_reset.adoc"

File: content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Introduction.adoc [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+
+The following YubiHSM software is used in this guide. These items are included as part of the archive file you download from the https://www.yubico.com/products/services-software/download/yubihsm-2-libraries-and-tools/[YubiHSM 2 libraries and tools page].
+
+
+=== Software
+[options="header"]
+|========================
+|Software                                                                                             |Purpose
+
+| https://developers.yubico.com/YubiHSM2/Component_Reference/yubihsm-connector/[YubiHSM Connector]    |Enables communication between the YubiHSM 2 and applications that use it. We recommend that the YubiHSM Connector run on the host operating system if the calling application is deployed to a VM. The Connector must always be running.
```
Comment (klali) [inline content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Introduction.adoc]: Please make all of these links relative (like: "link:../../Component_Reference/yubihsm-connector/"

File: content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Install_the_YubiHSM_Tools_and_Software.adoc [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+== Install the YubiHSM 2 Tools and Software
+
+To complete the procedures in this guide, install the YubiHSM 2 tools and software.
+
+[TIP]
+=====
+A generic prompt, *$*, is used in command line examples in this document. Depending on your command line application, your prompt may be different.
+=====
+
+*Step 1* Unzip the downloaded https://developers.yubico.com/YubiHSM2/Releases/[archives of the SDK] containing the YubiHSM libraries and tools and move the contents to an appropriate location.
```
Comment (klali) [inline content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Install_the_YubiHSM_Tools_and_Software.adoc]: Please make this link relative, like: "link:../../Releases"

File: content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Introduction.adoc [hunk 10]
```diff
@@ -0,0 +1,30 @@
...
+| https://developers.yubico.com/YubiHSM2/Component_Reference/yubihsm-shell/[YubiHSM Shell]            |The administrative command line tool used to interact with and configure the YubiHSM 2 device. If the YubiHSM Shell is installed on a VM, it will connect to the Connector over a networked connection.
+| https://developers.yubico.com/YubiHSM2/Component_Reference/yubihsm-setup/[YubiHSM Setup]            |Helps with setting up a device for specific use cases. Currently supports setting up for use with Microsoft Windows KSP.
+| https://developers.yubico.com/YubiHSM2/Component_Reference/KSP/[YubiHSM Key Storage Provider (KSP)] |Acts like a driver for the YubiHSM 2 device on Windows and enables it to work with applications that leverage Microsoft’s Cryptographic API Next Generation (CNG). Calling applications are for example Microsoft Certificate Services or Microsoft SQL Server Always Encrypted.
+|========================
+
+link:Prerequisites_and_Preparations.html[Next: Prerequisites and Preparations]
```
Comment (klali) [inline content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Introduction.adoc]: Please change this to be a link to the adoc file, that way it works for both adoc and html.

File: content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Install_the_YubiHSM_Tools_and_Software.adoc [hunk 11]
```diff
@@ -0,0 +1,60 @@
...
+
+*Step 6* You now have an administrative connection to the YubiHSM 2 and can list the objects available. To list the objects, type `list objects 0` and press Enter. Your results should be similar to the following:
+
+         `id: 0x0001, type: authentication-key, sequence: 0`
+
+         To exit, type `quit`.
+
+
+
+link:Configure_the_Primary YubiHSM_2_Device.html[Next: Configure the Primary YubiHSM 2 Device]
```
Comment (klali) [inline content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Install_the_YubiHSM_Tools_and_Software.adoc]: Please change this to be a link to the adoc file, then it works for both html and adoc.

File: content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Prerequisites_and_Preparations.adoc [hunk 12]
```diff
@@ -0,0 +1,36 @@
...
+
+image::privacy-threshold.png[]
+
+The YubiHSM Setup Tool enables you to perform the key splitting and assign shares to key custodians. If your organization has policies in place that define how this procedure should be performed, be sure you know these policies before proceeding.
+
+To complete the setup process, you need to know who the wrap key custodians will be. During setup, all key custodians must be physically present to record their shares. You should also have a predetermined practice in place specifying how the key shares must be recorded (written on paper, photographed, locally printed, or some other means) and how they must be stored between uses (for example, offsite archive, safety deposit box, sealed envelope).
+
+
+
+link:Install_the_YubiHSM_Tools_and_Software.html[Next: Install the YubiHSM 2 Tools and Software]
```
Comment (klali) [inline content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Prerequisites_and_Preparations.adoc]: Please change this to be a link to the adoc file, then it works for both html and adoc.

File: content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Prerequisites_and_Preparations.adoc [hunk 13]
```diff
@@ -0,0 +1,36 @@
+
```
Comment (klali) [inline content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Prerequisites_and_Preparations.adoc]: Please remove the blank line at the start of file.

File: content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Introduction.adoc [hunk 14]
```diff
@@ -0,0 +1,30 @@
+
```
Comment (klali) [inline content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Introduction.adoc]: Please remove the blank line at start of file.

File: content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Backup_Key_Material.adoc (hunk 1 above)
Comment (FFincannon) [reply content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Backup_Key_Material.adoc]: Um... what procedure should I follow to ensure that this PR is NOT merged before the document is completed and I have verified the formatting?

File: content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Configure_the_Primary YubiHSM_2_Device.adoc (hunk 2 above)
Comment (FFincannon) [reply content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Configure_the_Primary YubiHSM_2_Device.adoc]: I will remove them throughout.

File: content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Configure_the_Primary YubiHSM_2_Device.adoc (hunk 4 above)
Comment (FFincannon) [reply content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Configure_the_Primary YubiHSM_2_Device.adoc]: Done

File: content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Configure_the_Primary YubiHSM_2_Device.adoc (hunk 5 above)
Comment (FFincannon) [reply content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Configure_the_Primary YubiHSM_2_Device.adoc]: Changed (thank you for providing the link to replace it with)

File: content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Backup_Key_Material.adoc (hunk 1 above)
Comment (klali) [reply content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Backup_Key_Material.adoc]: You can change the title to start with "WIP:" (work in progress) to mark that you're still working on it.

File: content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Install_the_YubiHSM_Tools_and_Software.adoc (hunk 6 above)
Comment (FFincannon) [reply content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Install_the_YubiHSM_Tools_and_Software.adoc]: Done

File: content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Install_the_YubiHSM_Tools_and_Software.adoc (hunk 7 above)
Comment (FFincannon) [reply content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Install_the_YubiHSM_Tools_and_Software.adoc]: Done

File: content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Introduction.adoc (hunk 8 above)
Comment (FFincannon) [reply content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Introduction.adoc]: Done

File: content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Install_the_YubiHSM_Tools_and_Software.adoc (hunk 9 above)
Comment (FFincannon) [reply content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Install_the_YubiHSM_Tools_and_Software.adoc]: Done

File: content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/.conf.json (hunk 3 above)
Comment (FFincannon) [reply content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/.conf.json]: Thanks for letting me know exactly what was wrong

File: content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Introduction.adoc (hunk 10 above)
Comment (FFincannon) [reply content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Introduction.adoc]: Done throughout

File: content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Install_the_YubiHSM_Tools_and_Software.adoc (hunk 11 above)
Comment (FFincannon) [reply content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Install_the_YubiHSM_Tools_and_Software.adoc]: Done throughout

File: content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Prerequisites_and_Preparations.adoc (hunk 12 above)
Comment (FFincannon) [reply content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Prerequisites_and_Preparations.adoc]: Done

File: content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Prerequisites_and_Preparations.adoc (hunk 13 above)
Comment (FFincannon) [reply content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Prerequisites_and_Preparations.adoc]: Done

File: content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Introduction.adoc (hunk 14 above)
Comment (FFincannon) [reply content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Introduction.adoc]: Done

**Comment by FFincannon** [comment]
@sebastianelfors , this guide is now complete, so PR is ready for merging by @dainnilsson / @klali  / @emlun ...

**Pull request merged**